### PR TITLE
Clean up: tissue sample

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions
- establish alphabetical order

MAJOR changes:
none

SPECIAL ATTENTION:
- replaced linked schemas for `anatomicalLocation` by a new category called `anatomicalLocation` (Note: Category not added to the schemas yet, but the following schemas will receive this category:
     - sands/CustomAnatomicalEntity
     - sands/ParcellationEntity
     - sands/ParcellationEntityVersion
     - controlledTerms/Organ
     - controlledTerms/OrganismSubstance
     - controlledTerms/UBERONParcellation
     - controlledTerms/SubcellularEntity)

### CODE REVIEW

File: schemas/research/tissueSample.schema.tpl.json [hunk 1]
```diff
@@ -6,7 +6,16 @@
     "studiedState",
     "type"
   ],
-  "properties": {    
+  "properties": {        
+    "anatomicalLocation": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add all anatomical entities that describe the visible anatomy of this tissue sample.",
```
Comment (lzehl) [inline schemas/research/tissueSample.schema.tpl.json]: not sure about "... visible anatomy..." why not "... anatomical location..." ?

**Review by lzehl — APPROVED** [review]

**Comment by lzehl** [comment]
@apdavison or @Peyman-N or @aeidi89 feel free to merge if you also approve

**Review by lzehl — APPROVED** [review]

**Review by aeidi89 — COMMENTED** [review]
This looks good to me.
But the new category has not been added to the other schemas as of now.

**Comment by tgbugs** [comment]
Yes, ideally want `sands/ParcellationEntityVersion` whenever possible.

**Pull request merged**